e2e/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const { join } = require('path')
+
+const target = 'web'
+
+module.exports = {
+  mode: 'development',
+  target,
+  entry: {
+    plugin: './src/plugin.ts',
+    parent: './src/parent.ts',
+  },
+  output: {
+    filename: '[name].bundle.js',
+    chunkFilename: '[name].chunk.js',
+    path: __dirname + '/dist',
+  },
+  devtool: 'source-map',
```

Comment:
Can probably drop the source-map here (`: false`) to improve build times. If we aren't debugging this, the source-map isn't helpful.

Reply:
It's not measurably quicker if we remove source maps (the build is very small) so I'll leave them enabled for now.